pkg/api/frontendlogging/sentry.go

```diff
@@ -93,3 +94,16 @@ func (event *FrontendSentryEvent) ToLogContext(store *SourceMapStore) log15.Ctx
 
 	return ctx
 }
+
+func (event *FrontendSentryEvent) MarshalJSON() ([]byte, error) {
+	eventJSON, err := json.Marshal(event.Event)
+	if err != nil {
+		return nil, err
+	}
+	exceptionJSON, err := json.Marshal(map[string]interface{}{"exception": event.Exception})
+	if err != nil {
+		return nil, err
+	}
+	exceptionJSON[0] = ','
+	return append(eventJSON[:len(eventJSON)-1], exceptionJSON...), nil
+}
```

Comment:
Well, this was a funny Go puzzle I posted on Slack. Basically, Marshalling uses Sentry Event's "exception" field, while Unmarshalling uses FrontendSentryEvent's "exception" field. In other word, mockRequestBody+web.Bind wouldn't be able to pass the same object via JSON marshalling.

The solution was to override the MarshalJSON and stick it all together myself. Kind of works, but I shall talk to the original code authors about a possible rename of the fields.